cmd/psa/verify_as_attester_test.go

```diff
@@ -163,6 +163,31 @@ func Test_AttesterCmd_ok(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func Test_AttesterCmd_bad_nonce_Sz(t *testing.T) {
```

Comment:
no need to capitalise after an underscore:

s/Sz/sz/

Reply:
Done!